python/tests/unit/connectors/azure_ai_inference/conftest.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from unittest.mock import patch
+
+import pytest
+from azure.ai.inference.aio import ChatCompletionsClient, EmbeddingsClient
+from azure.ai.inference.models import ModelInfo, ModelType
+from azure.core.credentials import AzureKeyCredential
+
+from semantic_kernel.connectors.ai.azure_ai_inference import (
+    AzureAIInferenceChatCompletion,
+    AzureAIInferenceTextEmbedding,
+)
+
+
+@pytest.fixture()
+def azure_ai_inference_unit_test_env(monkeypatch, exclude_list, override_env_param_dict):
```

Comment:
Is there a specific reason we can't add the fixtures to our main unit test conftest.py file?

Reply:
We can. But since we can have nested `conftest` files, I thought I would group the azure_ai_inference specific ones in one file under the directory to avoid making the one on the root longer and longer.

Reply:
I think that's fine as-is, then. Later we could clean up the current conftest to make it less congested. Thank you!